ts/data.ts

```diff
@@ -1,6 +1,75 @@
-const data = {
+interface Data {
+  view: string;
+  entries: Entry[];
+  editing: null;
+  nextEntryId: number;
+}
+
+const data: Data = {
   view: 'entry-form',
   entries: [],
   editing: null,
   nextEntryId: 1,
 };
+
```

Comment:
This should be getting created by calling `readData`
```suggestion
const data = readData();
```